.travis.yml

```diff
@@ -4,6 +4,7 @@ language: python
 stages:
   - name: static analysis
   - name: test
+  - name: mypy_test
```

Comment:
Why does this require a separate stage and the doubling of the number of test jobs? Could this not be integrated into the existing test suite?

Reply:
No reason in particular.  Some projects define a separate test for type annotations, and some roll it into other jobs.  I wasn't sure what you'd want, so chose one to start with to demonstrate how we could include this in CI.  What would you like the travis setup to be?

Reply:
I see no reason to change the Travis setup. Just integrate the tests into the existing test suite.

Reply:
Done.